server/src/main/java/org/elasticsearch/index/mapper/TimeSeriesIdFieldMapper.java

```diff
@@ -67,7 +67,7 @@ public class TimeSeriesIdFieldMapper extends MetadataFieldMapper {
      * comfortable given that dimensions are typically going to be less than a
      * hundred bytes each, but we're being paranoid here.
      */
-    private static final int DIMENSION_VALUE_LIMIT = 1024;
+    private static final int DIMENSION_VALUE_LIMIT = 1536;
```

Comment:
I don't think we can increase this constant? The limit per field is DIMENSION_VALUE_LIMIT + DIMENSION_NAME_LIMIT. Otherwise we may spend up to ~2kb per field and then we hit the 32kb limit with less then 21 fields?

Reply:
Ok I missed that...the tsid includes both the name and the value.